ui/DesignSystem/Component/ProjectAnchorPopover.svelte

```diff
@@ -0,0 +1,145 @@
+    });
+  };
+
+  const openEtherscan = (txId: string) => {
+    org.openTxOnEtherscan(txId);
+    return undefined;
+  };
+
+  $: anchorColor =
+    anchor.type === "confirmed"
+      ? "--color-primary"
+      : "--color-foreground-level-5";
+  $: title = anchor.type === "confirmed" ? "Anchored in" : "Pending in";
+  $: titleColor =
+    anchor.type === "confirmed"
+      ? "--color-primary"
+      : "--color-foreground-level-5";
+</script>
+
+<style>
+  .container {
+    position: absolute;
+  }
+  .modal {
+    position: absolute;
+    width: 354px;
+    border-radius: 0.5rem;
+    background: var(--color-background);
+    box-shadow: var(--color-shadows);
+    top: -1rem;
+    left: 2rem;
+  }
+  .header {
+    border-bottom: 1px solid var(--color-foreground-level-2);
+    height: 3.5rem;
+    align-items: center;
+    display: flex;
+    padding: 0 46px 0 1rem;
+    color: var(--color-primary);
+  }
+  .meta {
+    display: flex;
+    color: var(--color-foreground-level-6);
+    margin: 1rem;
+    align-items: center;
+  }
+  .org {
+    color: var(--color-foreground-level-6);
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+</style>
+
+<Hoverable bind:hovering={hover} style="display: inline-flex;">
+  <Icon.AnchorSmall style={`fill: var(${anchorColor});`} />
+
+  <div class="container" on:click|stopPropagation>
+    {#if hover}
+      <div class="modal" out:fade|local={{ duration: 100, delay: 250 }}>
+        <div class="header">
+          <Icon.Anchor
+            style={`fill: var(${anchorColor}); margin-right: 0.5rem;`} />
+
+          {#if anchor.orgAddress}
+            <p class="typo-text-bold" style={`color: var(${titleColor})`}>
+              {title}
+            </p>
+            <Avatar
+              size="small"
+              style="margin: 0 0.5rem 0 0.5rem;"
+              variant="square"
+              avatarFallback={radicleAvatar.generate(
+                anchor.orgAddress,
+                radicleAvatar.Usage.Any
+              )} />
+            <p
+              class="typo-text-bold org"
+              style="color: var(--color-foreground-level-6);overflow: ellipsed">
+              {anchor.orgAddress}
+            </p>
+          {/if}
+        </div>
+        {#if anchor.type === "pending"}
+          <div class="meta">
+            <Icon.Pen style="margin-right: 0.5rem;" />
+            <p class="typo-text-small-bold" style="margin-right: 0.5rem;">
+              Signed by {anchor.confirmations} of {anchor.threshold}
+            </p>
+          </div>
+        {/if}
+
+        {#if anchor.type === "confirmed"}
+          <div class="meta">
+            <Icon.Ethereum style="margin-right: 0.5rem;" />
+            <p class="typo-text-small-bold" style="margin-right: 0.5rem;">
+              Transaction hash
+            </p>
+            <p
+              class="typo-text-small typo-link"
+              on:click={openEtherscan(anchor.transactionId)}>
```

Comment:
This only works as expected due to a svelte bug. To fix this do
```suggestion
              on:click={bindOpenEtherscan(anchor.transactionId)}>
```

where
```typescript
const bindOpenEtherscan = (txId: string) => () => {
  org.openTxOnEtherscan(txId);
};
```

Reply:
2bba75b2b87bce66f2a37f94a5303ebd744943ed